src/app/user-sign-up/register/register.component.ts

```diff
@@ -26,20 +32,77 @@ export class RegisterComponent implements OnInit {
     })
   }
 
-  // getter methods
+  /**
+   * name method are used to get name
+   * @retun name
+   *  */
   get name(){
     return this.registerForm.get('name');
   }
+  /**
+   * email method are used to get email
+   * @retun email
+   *  */
   get email(){
     return this.registerForm.get('email');
   }
+  /**
+   * password method are used to get password
+   * @retun password
+   *  */
   get password(){
     return this.registerForm.get('password');
   }
-
-  // Form submit method
+  /**
+   * addUser method are used to subscribe data
+   * @param new_user : User_register
+   *  */ 
+  addUser(new_user: User_register) {
+    this.add_users.addUsers(new_user).subscribe(
+      (x) => {
+        alert("Registration Succesful");
+        this.router.navigate(['/login']);
+      },
+      (error: HttpErrorResponse) => {
```

Comment:
Why this error codes are written here?

Reply:
@HarishChoure waiting for reply here

Reply:
Error code are written here for showing alert to the user, If user entering wrong details and If any server error in the backend.